charts/unpoller/templates/dashboards.yaml

```diff
@@ -0,0 +1,78 @@
+{{- if .Values.dashboards.create }}
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDashboard
+metadata:
+  name:  unpoller-uap-insights
+  labels:
+    {{- include "unpoller.labels" . | nindent 4 }}
+spec:
+  instanceSelector:
+    matchLabels:
+      {{- include "unpoller.dashboadSelectorLabels" . | nindent 6 }}
+  url: "https://grafana.com/api/dashboards/11314/revisions/10/download"
```

Comment:
Probably don't want the revision in these urls.

Reply:
I agree, but I could not find a URL to download it that did not include the revision:
`https://grafana.com/api/dashboards/11314/download` => 404

If you have maybe the source of the repos I can point to them?

Reply:
`https://grafana.com/api/dashboards/11314/revisions/latest/download`

Reply:
:facepalm:  Thanks, I'll update it.

Reply:
fixed.